### PR TITLE
Adding rapid-pe package

### PR DESCRIPTION
**Pull request by dwysocki**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dwysocki** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/rapid-pe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - rapid_pe
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://git.ligo.org/rapidpe-rift/rapidpe/
+  summary: 'RapidPE: The original low-latency gravitational wave parameter estimation code.'
+  license: GPL-2.0
```
Comment (duncanmmacleod) [inline recipes/rapid-pe/meta.yaml:44]: @dwysocki, this is ambiguous, can you change this to one of

```suggestion
  license: GPL-2.0-only
```

or

```suggestion
  license: GPL-2.0-or-later
```

Comment (dwysocki) [reply recipes/rapid-pe/meta.yaml]: Thanks for catching this, I've taken your latter suggestion in the latest commit

**Comment by dwysocki** [comment]
@conda-forge-admin, please ping conda-forge/staged-recipes

File: recipes/rapid-pe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rapid_pe-{{ version }}.tar.gz
+  sha256: 53739bd328779f8484cfc1887ba18aa7b8a0b0ea82b064bb4866b2cbc6eaa518
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/rapid-pe/meta.yaml:15]: If you are skipping Windows just b/c the lack of pytorch then you should still make this package `noarch` and let the solver fail on Windows. Then when pytorch is available on Windows, this package will be ready.

Comment (dwysocki) [reply recipes/rapid-pe/meta.yaml:15]: It's not PyTorch, but rather `lal` which doesn't support Windows (see [here](https://github.com/conda-forge/lal-feedstock/blob/df3c97bc502f2d24e1b3fdf8f4ef9f393abbd757/recipe/meta.yaml#L37)).    I simply copied the `skip` line from `lal`'s `meta.yaml`, though I'm happy to just let the pipeline fail if that's the preferred approach.

Also it looks like the executables would not be made accessible on Windows conda/conda-build#4385, which would make our package not very useful.

Comment (ocefpaf) [reply recipes/rapid-pe/meta.yaml:15]: `lal` is also unavailable on Windows but that one cannot be noarch, this one can. So the situation is the same. However, if this package uses the old and depreacated scripts, then you are correct, this cannot be noarch :-/

**Pull request merged**